### PR TITLE
Add the implementation for the census map on Mapbox

### PR DESCRIPTION
**Pull request by bethanyaconnor**
This will be hidden behind an experiment flag to begin with.

### CODE REVIEW

File: apps/src/templates/census2017/CensusMapReplacement.jsx [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+      });
+      _this.map.addLayer({
+        id: 'census-schools-teaching-cs',
+        type: 'symbol',
+        source: 'censustiles',
+        'source-layer': 'census',
+        layout: {
+          'icon-image': 'marker-15-green',
+          'icon-allow-overlap': true,
+          'icon-size': ['interpolate', ['linear'], ['zoom'], 1, 0.7, 14, 1.6]
```
Comment (breville) [inline apps/src/templates/census2017/CensusMapReplacement.jsx:249]: Did we choose these numbers, and are they meaningful in some way specific to us?

**Review by breville — APPROVED** [review]
I gave this a faster scan than usual, and it held up to that :)

File: apps/src/templates/census2017/CensusMapReplacement.jsx (hunk 1 above)
Comment (bethanyaconnor) [reply apps/src/templates/census2017/CensusMapReplacement.jsx:249]: I chose them. I'll add a comment explaining what they do, though the numbers themselves are just based on what looked alright to me.

File: apps/src/templates/census2017/CensusMapReplacement.jsx [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+  }
+}
+
+export default class CensusMapReplacement extends Component {
+  static propTypes = {
+    onTakeSurveyClick: PropTypes.func.isRequired,
+    school: PropTypes.object
+  };
+
+  map = undefined;
```
Comment (breville) [inline apps/src/templates/census2017/CensusMapReplacement.jsx:136]: Should these all have `var`?

Comment (bethanyaconnor) [reply apps/src/templates/census2017/CensusMapReplacement.jsx:136]: Honestly not sure but they don't in the previous implementation: https://github.com/code-dot-org/code-dot-org/blob/15823fe1bf61b0f071c89995cefe0ee23edfed04/apps/src/templates/census2017/CensusMap.jsx#L137

**Pull request merged**